string nodeName
bool remove
---
SystemStats[] result
